### ISSUE TITLE
Austin Apartment Complex Safety  

### ISSUE THREAD

**Issue by TheSecMaven** [open-austin/project-ideas#106]
### What problem are we trying to solve?
College students can have a hard time deciding which apartment complex is best for them. Things like crime in the surrounding area and nearby construction can play a factor in their decision. Often, it is hard to visualize this data and even harder to compare it to apartment complexes nearby.

We can pull the list of apartments in Austin from google. Then, using open crime and building permit data, overlay it with crimes in the surrounding area. This would allow the user to visualize nearby crime relative to an apartment complex.

### Who will benefit (directly and indirectly) from this project?
Anybody searching for an apartment in Austin, students and more, will be able to see how safe the area around the apartment complex they choose is. It will be readily available for the average user to use, and won't require tons of research. 

Not only this, but when I came up with this idea the goal was to hold apartment complexes more accountable for their surrounding area. They can implement more security measures if they are necessary. With this tool, the crime around an apartment becomes public knowledge that is easily viewable. If apartment buyers see how had a complex is, they won't live there. When complexes realize why people are not living there, they will seek to solve the problem. In this way, we will benefit the community. 

### Where can we find any research/data available/articles?
I have done this project at my local school hackathon. It was not as advanced as I am hoping to make it. https://github.com/mkkeffeler/CrimsonHacks17

I replicated and improve upon this project for the city of Memphis. Used google maps to get a list of apartment complexes, rather than webscraping. Also made it more automated, so that updates can be pushed more easily. https://github.com/mkkeffeler/OurMemphis (Check it running here: http://ourmemphis.azurewebsites.net/ NOTE: the heatmap does not work yet in this version)

### What help is needed at this time?
Developers who want to implement more features. I want to create a search function for the apartment complexes, and also would like to add the ability to add your own apartment complex. I have more ideas on what to add for capabilities but one day at a time. 

### What are the next steps (validation, research, coding, design)?
Research and design. I want to get others opinions on more things that could be added. 311 call data? I am open to ideas. After research, I want to redesign the interface. Currently using toggle switches as buttons, which can be improved and made cleaner.

----

#### Checklist for NEW ideas :baby:
Hey, you're official! You're now part of the growing civic hacking community in Austin. Here's a few things to get started (a couple you've probably already done).

- [x] Create this idea issue
- [x] Flesh out the who, where, and what questions above
- [x] Start the conversation about this new idea on [Slack](https://openaustin-slackin.herokuapp.com/)!
- [x] Respond with at least one update on this issue within the next month

#### Checklist for ACTIVE projects :fire:
Let's get this project started! When this idea starts taking off, the Projects Core Team will start helping this project's lead(s) out with project management and connecting you to resources you may need. To get there, please complete and check off the following:

- [ ] Complete [Open Leadership 101](https://mozilla.teachable.com/p/open-leadership-101)
- [ ] Create a [GitHub repository](https://github.com/new) and Slack channel for work.
- [ ] Create a [README file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/) in your project repository. This file should help newcomers understand what your project is, why it's important, and kinds of help you're looking for.
- [ ] [Create issues](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/project-set-up-for-collaboration-with-github/#assignment--add-your-first-issue) to describe each task that you plan to do or need help with and how a contributor can get started on that task. You might start and stop a lot, so consider issues as your to-do list.

#### Checklist for FEATURED Projects :tada:
To have your project FEATURED on [Open-Austin.org](https://open-austin.org/), complete the following documentation. In past projects, well-documented featured projects have more contributions than other projects.

* [ ] In your README, link to the [Open Austin Community Participation Guidelines](https://docs.google.com/document/d/1OujyBccPpepUXSY5_nP3alzvd81WlCKSeSf_-sU5K-U/edit?usp=sharing) or write your own code of conduct.
* [ ] [Create file: LICENSE](http://choosealicense.com/) to give your project an open license, allowing for sharing and remixing.
* [ ] [Create file: CONTRIBUTING.md](https://github.com/acabunoc/mozsprint-repo-template/blob/master/CONTRIBUTING.md) so others know [how they can contribute](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-contributor-guidelines/).

Once all of the above is complete,
- [ ] [Create an issue](https://github.com/open-austin/open-austin.github.io/issues/new) on the open-austin.github.io repo with the title `Add [this project] to projects page. An Open Austin leader will review this issue and post your project :balloon:

If you get stuck at any point, feel free to reach out to the leadership team on Slack by adding @leadership to your message. We're here to help you make real changes to our city.

**Comment by maeganjwilson** [comment]
I'm interested in helping with this. I haven't been to an open austin event yet cause of my job and traveling, but I'm wanting to help and get involved!

**Comment by mkkeffeler** [comment]
@meaganjwilson Excellent! I will reach out to you Via slack and we can get things started. Thanks so much!

**Comment by wittrura** [comment]
I would also be very interested in helping out with this! Also haven't been to an event yet, but I wanted to look over some possible projects first and this caught my eye.  I've been doing some digging around and it seems like the crime data may pose a bit of a challenge but I'm for seeing what's possible.

**Comment by Woodley** [comment]
I also want to be involved want to see data from complaints sent to Austin
Tenant Council, Fair Housing, ARC, Echo, Adapt of Texas, Rio Grande Legal
Aid, Disability Right Texas, Texas Civil Rights Project, American Civil
Liberties Union, Austin Housing Authority, HUD, Realtors Association, Texas
Department of Licensing and Regulations, Fire Marshals, etc..

In my mind the word safety includes building codes, Crime, maintenance,
fire, pools, lighting, accessibility, single or multiple egress/ingress,
etc..

**Comment by werdnanoslen** [comment]
Hi y'all, I'm the projects lead, so if you need anything for this let me
know. Can y'all make it to our hack night? Check Meetup for details

**Comment by mkkeffeler** [comment]
Hey everyone,

So glad to see we have a number of people interested in this project! 

I will be starting a channel for discussion about this project in the Open Austin slack right now. 

I am going to name **the slack channel for this project "p-apartmentsafety"** 

If everybody could please put in the slack what you have worked with in the past so that we can start to get an idea of who is going to do what exactly that would be awesome. We can also hash out some possibilities for the project outside of the initial idea as it seems some of you may have additional ideas, which is awesome!

If somebody is having issues or anything of the like please comment here and we can go from there. 

Thanks all!

**Comment by werdnanoslen** [comment]
Nice! You've completed all the steps we'd like to see in a new project, so I'll flag this one as an active project. See the checklist in the first post for some things that might help y'all out, and mention/message me here or Slack for help. Good work y'all!

**Comment by 2echoi** [comment]
Hello! I am UX designer recently graduated this May. I'm interested in joining this project but are you guys looking for UX designer at this time?

**Comment by werdnanoslen** [comment]
Hi @2echoi, you can also join our Slack team at [slack.open-austin.org](slack.open-austin.org), and then join the #p-apartmentsafety channel

**Comment by anu007** [comment]
Hello, I am Anu and have a degree in UX Design.I wanted to join the team, Please let me know do you guys need UX Designer(Researcher)?I am open to work on other projects.Thanks.

**Comment by mkkeffeler** [comment]
Hello @anu007 We do have a number of people who have expressed interest in working on the UX. Please join the slack channel for this project  at[ slack.open-austin.org](slack.open-austin.org), and then join the #p-apartmentsafety channel!

**Comment by anu007** [comment]
Hello @mkkeffeler.Thanks for sharing the link for Slack group.
